thread/thread.cpp

```diff
@@ -1090,6 +1090,12 @@ R"(
         }
         return photon::now;
     }
+    void alog_update_now(uint64_t *sec, uint64_t *usec) {
+        last_tsc = _rdtsc();
+        auto ts = update_now();
+        *sec = ts / 1000000;
+        *usec = ts % 1000000;
+    }
     int timestamp_updater_init() {
```

Comment:
Why not move to alog.cpp?

Reply:
`update_now()` `_rdtsc()` `last_tsc` are defined as inline or static unit in thread.cpp, `alog_update_now()` needs to call those functions and set `last_tsc` `now` so time update can also used by thread scheduler.